e2e/test/scenarios/permissions/admin-permissions.cy.spec.js

```diff
@@ -581,6 +581,51 @@ describe("scenarios > admin > permissions", () => {
       cy.findByText("Data permissions");
     });
   });
+
+  it("should show a dismissable modal and banner showing split permermisson changes (#metabase#45073", () => {
+    const tempState = {
+      "show-updated-permission-modal": true,
+      "show-updated-permission-banner": true,
+    };
+
+    cy.intercept("/api/session/properties", req => {
+      req.continue(res => {
+        res.body = { ...res.body, ...tempState };
+      });
+    }).as("sessionProps");
+
+    cy.intercept("api/setting/show-updated-permission-modal", () => {
+      tempState["show-updated-permission-modal"] = false;
+    });
+
+    cy.intercept("api/setting/show-updated-permission-banner", () => {
+      tempState["show-updated-permission-banner"] = false;
+    });
```

Comment:
@npfitz can you add a brief comment on what this is doing? it's maybe not obvious that there's a second get request happening and you're altering the state at the time of these requests.